src/leiningen/new/cryogen/themes/blue/html/base.html

```diff
@@ -95,5 +95,7 @@
 <script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.0/js/bootstrap.min.js"></script>
 {% script "js/highlight.pack.js" %}
 <script>hljs.initHighlightingOnLoad();</script>
+{% if post.klipse %} {{post.klipse|safe}} {% endif %}
+{% if page.klipse %} {{page.klipse|safe}} {% endif %}
```

Comment:
Code blocks looks a little strange when both hljs and klipse are applied in conjunction. Perhaps hljs should be disabled if klipse is enabled.